docs/CONAN_USER_DOCUMENTATION.md

```diff
@@ -1,3 +1,14 @@
-# Documentation
+# How to use Conan with NXRM
 
-Coming soon!
+1. Install Conan (if on OSX you can do this via ```brew install conan```)
+2. Create the conan repositories in NXRM (e.g. conan-hosted)
+3. Add the conan security realm to NXRM: Login as an admin, go to settings > security > realms and add 
+   ```Conan Bearer Token Realm```
+4. (Optional) Conan will already have a remote configured so if you want it to exclusively use NXRM you will need to remove that 
+   remote first via ```conan remote remove conan-center```
```

Comment:
NOTE: In NuGet, if you don't do something like this, is can pull from nuget.org first.  If Conan is similar, may want to denote that.  e.g. "If you don't do this, then you may inadvertantly bypass NXRM when pulling packages."

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/28/commits/a70e3250e2a5de9f8d5a1e0145d1a6f8d000626a